iFixit/pulldasher
# Deps: upgrade all the things!

* Drop unused deps
* Upgrade all deps as far as they'll go and still be compatible
  * chakra doesn't have full react-18 support yet

### QA
Make sure the app runs and the frontend works... click "refresh" and ensure the pull blinks in the new UI.

cr_req 1 (not much to CR here)
